Products/Commodiry.UnitTests/CommodityUnitTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Commodiry;
```

Comment:
Исправьте опечатку (после добавления в проект ссылки на библиотеку).
```suggestion
using Commodity;
```